projects/igniteui-angular/src/lib/date-range-picker/date-range-picker.component.ts

```diff
@@ -790,46 +798,57 @@ export class IgxDateRangePickerComponent extends DisplayDensityBase
         });
     }
 
+    private parseMinMax(value: string | Date, isStart = true): Date | null {
```

Comment:
please not.
What this method does 👀 
please refactor this. You may have two methods `parseMin` and `parseMax`. If both methods share some logic move it to additional method, e.g. `parseDate`